writer_test.go

```diff
@@ -565,3 +571,85 @@ func TestBloomFilterForDict(t *testing.T) {
 		t.Error("bloom filter should have contained 'test'")
 	}
 }
+
+func TestEmptyFile(t *testing.T) {
+	f := func(t *testing.T, schema *parquet.Schema) {
```

Comment:
`t.Helper()`?

Reply:
Sorry, how is t.Helper will help here ?

Reply:
Point out the actual failing test, but I guess this is alright as is.